src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings

```diff
@@ -2810,7 +2810,7 @@ Die Angaben werden statistisch ausgewertet, sie werden nicht zu einem Profil ges
 
 /* Recovery Certificate Delete Alert */
 
-"RecoveryCertificate_Alert_title" = "Wollen Sie das Genesenenzertifikat wirklich entfernen?";
+"RecoveryCertificate_Alert_title" = "Wollen Sie das Zertifikat entfernen?";
```

Comment:
This is inconsistent with the other titles above

Reply:
Hi Nick, yes you are right. We'll think of writing 'Wollen Sie das Zertifikat entfernen?' for all occurrences, so that it's consistent. But here we have an example of a shorter text. https://github.com/corona-warn-app/cwa-app-ios/blob/release/2.5.x/src/xcode/ENA/ENA/Resources/Localization/de.lproj/Localizable.strings#L2655

We try always to adjust both Android and iOS texts. If the texts are different and we have to change them in the future, it's not so easy to find these texts. And besides, tester care a lot about the text consistency in both Ios and Android and if there are differences, the bugs will be opened.

Reply:
@nickguendling Would it be okay like this: 'Wollen Sie das Genesenenzertifikat entfernen?' ?